tests/ext2/plain/ext2.c

```diff
@@ -273,8 +283,8 @@ int main(int argc, const char* argv[])
         assert(buf.st_size == sizeof(alpha));
         assert(buf.st_mode == mode);
         assert(buf.st_nlink == 1);
-        assert(buf.st_uid == 0);
-        assert(buf.st_gid == 0);
+        assert(buf.st_uid == myst_syscall_geteuid());
```

Comment:
Why don't we use `getuid` and `getgid` directly?

Reply:
It will work either way but I switched to use to shorter forms.